src/components/ProfilePicture.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, BoxProps, Skeleton } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { fetchProfilePicture } from '../api/cristinApi';
+
+interface ProfilePictureProps {
```

Comment:
for å unngå å "kopiere" prop types, `sx` har vel en egen type `SxProps`, som ikke brukes her nå 
```suggestion
interface ProfilePictureProps extends Pick<BoxProps, "sx"> {
```